src/ert/shared/share/ert/forward-models/res/script/ecl_config.py

```diff
@@ -203,8 +206,38 @@ class FlowConfig(EclConfig):
 
     def __init__(self):
         config_file = os.getenv("FLOW_SITE_CONFIG", default=self.DEFAULT_CONFIG_FILE)
+        if not Path(config_file).exists():
+            config_file = self.init_flow_config()
         super().__init__(config_file, simulator_name="flow")
 
+    @staticmethod
+    def locate_flow_binary() -> str:
+        """Locate the path for a flow executable.
+
+        Returns the empty string if there is nothing to be found in $PATH."""
+        rhel_version = "7"
+        if "el8" in platform.release():
+            rhel_version = "8"
+        candidates = ["flow", f"/project/res/x86_64_RH_{rhel_version}/bin/flowdaily"]
```

Comment:
Do we know that the user raising [this issue](https://github.com/equinor/ert/issues/8223) is on "our" NFS?

Reply:
I don't think he is a part of the Equinor organization so I don't think he is from "our" NFS.

Reply:
I was wondering this because if this PR aims to solve the issue by the user (who is outside of our NFS), will this resolve it?

Reply:
Yes as long as flow is installed.